install.sh

```diff
@@ -521,6 +536,90 @@ do_install() {
 			echo_docker_as_nonroot
 			exit 0
 			;;
+			
+		sles)
+			arch=$(uname -m)
+			if [[ $arch != "s390x" ]]; then
+				echo "Docker-CE is not supported on SLES for $arch!!!"
+				exit 1
+			else
+				echo "Installing Docker-CE on $arch!"
+			fi
+
+			sles_repo="$DOWNLOAD_URL/linux/$lsb_dist/$REPO_FILE"
+			opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/SLE_15_SP2/security:SELinux.repo"
+			if ! curl -Ifs "$sles_repo" > /dev/null; then
+				echo "Error: Unable to curl repository file $sles_repo, is it valid?"
+				exit 1
+			fi
+			pre_reqs="ca-certificates curl libseccomp2 iptables awk"
+			
+			(
+				if ! is_dry_run; then
+					set -x
+				fi
+				$sh_c "zypper install -y $pre_reqs"
+				$sh_c "zypper addrepo $sles_repo"
+				
+				
+				cat >&2 <<-'EOF'
+						Warning !!! 
+						"Opensuse Repository (https://download.opensuse.org/repositories/security:SELinux) will be enabled now.
+						Do you wish to continue?
```

Comment:
Looks like there's a stray quote here. Also, I think the proper capitalisation of `Opensuse` is `openSUSE` (lowercase "open", all-caps "SUSE" - see https://www.opensuse.org)

```suggestion
						openSUSE repository (https://download.opensuse.org/repositories/security:SELinux) will be enabled now.
						Do you wish to continue?
```

Reply:
Updated the script with the suggested change.